app/views/auth_source_ldaps/_form.html.erb

```diff
@@ -24,13 +24,15 @@
                       :data => { :default_ports => AuthSourceLdap::DEFAULT_PORTS } %>
       <%= select_f f, :server_type, AuthSourceLdap::SERVER_TYPES, :first, :last,
                     { :include_blank => _("Choose a server type") },
-                    { :label => _('Server type') } %>
+                    { :label => _('Server type'), :onchange => 'change_ldap_server_type(this)' } %>
     </div>
     <div class="tab-pane" id="account">
       <%= text_f f, :account, :help_inline =>_("Use this account to authenticate, <i>optional</i>").html_safe %>
       <%= password_f f, :account_password, :help_inline => _("Use this account to authenticate, <i>optional</i>").html_safe, :unset => action_name == "edit" %>
       <%= text_f f, :base_dn, :label => _("Base DN"), :size => "col-md-8" %>
       <%= text_f f, :groups_base, :label => _("Groups base DN"), :size => "col-md-8" %>
```

Comment:
Would you mind adding `:label_help` with examples of group_base value for supported servers including netgroups? Good examples can be found at https://theforeman.org/manuals/1.15/#4.1.1LDAPAuthentication If you modify that, adding same help to other fields such as base_dn and account would be awesome.

Also updating the manual with netgroups base DN in manual would be appreciated.